pipeline/outputs/elasticsearch.md

```diff
@@ -115,6 +120,45 @@ In your main configuration file append the following _Input_ & _Output_ sections
 
 ![example configuration visualization from calyptia](../../.gitbook/assets/image%20%282%29.png)
 
+### Configuration File with Upstream
+
+In your main configuration file append the following _Input_ & _Output_ sections.
```

Comment:
```suggestion
In your main configuration file append the following `Input` and `Output` sections:
```

Reply:
I would prefer all cases like this to be fixed at once in a pull request dedicated to re-formatting.